cgmanifests/submodules/cgmanifest.json

```diff
@@ -1,6 +1,50 @@
 {
   "Version": 1,
   "Registrations": [
+    {
+      "Component": {
```

Comment:
@snnn did we update the thirdpartynotices file for these new deps?

Reply:
No, we don't ship them.